ai_ta_backend/vector_database.py

```diff
@@ -298,6 +298,8 @@ def _ingest_single_py(self, s3_path: str, course_name: str):
             'url': '',
             'base_url': '',
         } for doc in documents]
+      #print(texts)
+      os.remove(file_path)

```

Comment:
Should this `os.remove()` be here twice? Seems like it could cause bugs.

Reply:
Fixed this!